doc/user/content/integrations/sql-clients.md

```diff
@@ -79,4 +79,16 @@ Download and install the [PostgreSQL installer](https://www.postgresql.org/downl
 
 To connect to Materialize using [DBeaver](https://dbeaver.com/docs/wiki/), follow the documentation to [create a connection](https://dbeaver.com/docs/wiki/Create-Connection/) and use the **PostgreSQL database driver** with the credentials provided in the UI for your Materialize region.
 
-<img width="1314" alt="Screenshot 2022-12-24 at 18 32 20" src="https://user-images.githubusercontent.com/23521087/209447654-11a51e45-b68f-4e11-8e82-9036d8f7aed8.png">
+<img width="1314" alt="DBeaver Materialize Connection Details" src="https://user-images.githubusercontent.com/23521087/209447654-11a51e45-b68f-4e11-8e82-9036d8f7aed8.png">
+
+## DataGrip
+
+To connect to Materialize using [DataGrip](https://www.jetbrains.com/help/datagrip/connecting-to-a-database.html), follow the documentation to [create a connection](https://www.jetbrains.com/help/datagrip/connecting-to-a-database.html) and use the **PostgreSQL database driver** with the credentials provided in the UI for your Materialize region.
+
+<img width="1131" alt="DataGrip Materialize Connection Details" src="https://user-images.githubusercontent.com/21223421/217556108-e61ff518-a475-489a-b382-c8caf4fecbf4.png">
```

Comment:
Did the connection work with Database: `postgres`?

Reply:
Good point! It works with both, but I guess it's better to be consistent and use `materialize`. I've updated the screenshot!